channels/telegram/src/main/kotlin/com/justai/jaicf/channel/telegram/TelegramReactions.kt

```diff
@@ -42,7 +46,7 @@ class TelegramReactions(
         replyToMessageId: Long? = null,
         replyMarkup: ReplyMarkup? = null
     ) = sendMessage(text, parseMode, disableWebPagePreview, disableNotification, replyToMessageId, replyMarkup).also {
```

Comment:
We register reaction after `sendMessage` and `sendMessage` register reaction by itself, so reactions are registered twice.
The same for other types of telegram reactions